### ISSUE TITLE
mailtrain.org / docs.mailtrain.org are down

### ISSUE THREAD

**Issue by anarchist912** [Mailtrain-org/mailtrain#1170]
Have been down for at least the last 2 hours.

This software sounds so promising, but I can't get to the website.

**Comment by forgeOneHQ** [comment]
@anarchist912 what do you need? Most infos are documented directly here. :)

Gonna check with @bures to find a solution.

**Comment by anarchist912** [comment]
@talheim-it Thank you a lot for the quick reply. I was hoping to find some docs about the API, to see in advance, if I'll be able to do, what I need to do. I tested with [listmonk](https://github.com/knadh/listmonk), but it's not far enough yet.

I'll need to segment 100.000 to 1 Million of subscribers according to tags / attributes and also get a list of segments with subscriber count. Only successful double opt-ins should be counted. 

I also need to trigger a certain email when a certain count of subscribers is reached.

Will it be possible, what you think?
Luckily it is based on node, so i'll find my way around the routes quickly and will be able to add some code, if necessary.

I'd love to document, what I find, but I have too much pressure to get it done. I might quite possibly come back in the future, when there is more time, as it seems to be a very helpful tool for future tasks, as well.

PS: Do I have to set up the double opt-in via automation triggers, or is it a standard? In listmonk, if I set a list (=segment) to double opt-in, subscribers added via API will get a double opt-in mail and further.
